lib/features/pokemon_overview/pokemon_overview_page.dart

```diff
@@ -10,24 +11,40 @@ class PokemonOverviewPage extends StatelessWidget {
     Key? key,
   }) : super(key: key);
 
-  final List<Pokemon> pokemons;
+  final Async<List<Pokemon>> pokemons;
 
   @override
   Widget build(BuildContext context) {
+    void showNoPokemonAvailable(errorMessage) {
+      final snackBar = SnackBar(
+        content: Text('$errorMessage'),
+      );
+      WidgetsBinding.instance.addPostFrameCallback((_) {
```

Comment:
do you still need this `addPostFrameCallback`, seems not anymore since context is already available here as given from this build method 🙂